packages/client/src/utils/markedMathExtension.ts

```diff
@@ -0,0 +1,50 @@
+import type { TokenizerExtension } from 'marked';
+import type { PreviewMath } from '../config';
+
+const rules = {
+  inlineMathStart: /\$.*?\$/,
+  inlineMath: /^\$(.*?)\$/,
+  blockMath: /^(?:\s{0,3})\$\$((?:[^\n]|\n[^\n])+?)\n{0,1}\$\$/,
+};
```

Comment:
Through this is good, but the long words will remain after minify, so please split them or use shorter property as keys, thanks.😉